android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/ServiceCard.kt

```diff
@@ -170,8 +171,8 @@ private fun RowScope.RenderDetails(
   navController: NavController,
   resourceData: ResourceData,
 ) {
-  val iconsSplit = serviceMemberIcons?.split(",") ?: listOf()
-  val twoMemberIcons = iconsSplit.map { it.capitalize().trim() }.take(2)
+  val iconsSplit = serviceMemberIcons?.split(",")?.filter { it.isNotEmpty() } ?: listOf()
+  val nMemberIcons = iconsSplit.map { it.capitalize().trim() }.take(NUMBER_OF_ICONS_DISPLAYED)
```

Comment:
Rename to
```suggestion
  val memberIcons = iconsSplit.map { it.capitalize().trim() }.take(NUMBER_OF_ICONS_DISPLAYED)
```

Reply:
3 icons couldn't fit so I had to wait for guidelines on new icon sizes and @HenryRae did provide them. However, since the same render function is used on both screens, it enforces that same limit on both screens. But @HenryRae informed me that the household profile should have only 1 icon according to the designs even if more than one icon applies to a patient. For this reason, there may be need for a new render function for the household profile or some sort of enforcement where only one icon is passed to a patient profile based on some sort of hierarchy

Reply:
The icon displayed on the profile should be controlled by the rule defined to return the icon for re-usability that's the only way to overcome that issue and I think most profile configurations have been doing so, if not the issue can be resolved at configuration level.

Reply:
We can leave the limit to 2 icons; no problem.

Reply:
I agree with @ellykits I don't think we need to change the functionality to limit the number of icons on the profile. That can be handled on the configs plus there is a possibility for a person to have more than one icon on the profile. e.g if we have an icon for HIV and the same patient is pregnant then they will have 2 icons even on the profile.

Reply:
If we are limiting the icons to 2 then the counter should show if the icons are more than 2.